lib/modules/manager/flux/extract.ts

```diff
@@ -197,6 +207,29 @@ function resolveResourceManifest(
         deps.push(dep);
         break;
       }
+      case 'OCIRepository': {
+        const container = resource.spec.url?.replace('oci://', '');
+        const dep: PackageDependency<FluxManagerData> = {
+          depName: container,
+        };
+        if (resource.spec.ref?.digest) {
+          dep.currentDigest = resource.spec.ref.digest;
+          dep.currentValue = 'latest';
+          dep.datasource = DockerDatasource.id;
+        } else if (resource.spec.ref?.tag) {
+          const containerTag = getDep(container + ':' + resource.spec.ref?.tag);
```

Comment:
```suggestion
          const containerTag = getDep(container + ':' + resource.spec.ref.tag);
```
use template string instead of concatenation

Reply:
Fixed in upcoming commit